README.md

```diff
@@ -163,6 +163,15 @@ $ dm pkg import-all <path> <data_source>
 $ dm pkg import-all app/data/DemoApplicationDataSource DemoApplicationDataSource
 ```
 
+
+Exporting document(s) in a package as a zip file
```

Comment:
```suggestion
Exporting documents as a zip file
```
Should behave the same for all type of entities.
Also, should we unpack the zip file for the user? Perhaps not...